ui-tests/cypress/support/pageObjects/content/templates/TemplateForm.js

```diff
@@ -88,6 +82,7 @@ export default class TemplateForm extends AdminContent {
       cy.realPress(['Shift', 'ArrowLeft']);
     }
     cy.realPress(['Backspace']);
+    return cy.get('@currentPage');
   }
 
   editFormFields(payload) {
```

Comment:
same; but I understand is too elaborate to waste time refactoring it without specific requirements on the section 👌🏼